packages/create-app/templates/template-tutorial-todo-app/src/mocks.ts

```diff
@@ -18,7 +18,7 @@ const projects: MockProject[] = [
     $apiName: "MockProject",
     $primaryKey: "1",
     id: "1",
-    name: "Fake Project",
+    name: "Mock Project",
```

Comment:
Consistent capitalization given you changed the other project name.

```suggestion
    name: "Mock project",
```